packages/backend-common/package.json

```diff
@@ -19,14 +21,16 @@
   ],
   "license": "Apache-2.0",
   "scripts": {
-    "build": "backstage-cli build-cache -- tsc",
+    "build": "backstage-cli build --outputs cjs,types",
```

Comment:
Maybe `backend:build` here as well?

Reply:
It's not a backend though, so if we end up with separate tooling for backends this would probably be excluded. Basically any library-type thing is being built with build.